pkg/controller/extension/actuator_test.go

```diff
@@ -247,6 +250,52 @@ var _ = Describe("GenerateInstallManifest", func() {
 	})
 })
 
+var _ = Describe("ReconcileShootInfoConfigMap", func() {
+	var (
+		shootClient client.Client
+		config      *fluxv1alpha1.FluxConfig
+	)
+
+	It("should apply successfully and contain expected keys", func() {
+		shootName := "test-shoot"
+		technicalID := fmt.Sprintf("shoot--asdf-test-%s", shootName)
+		clusterIdentity := "magic-cluster-identity"
+		shootClient = newShootClient()
+		config = &fluxv1alpha1.FluxConfig{
+			Flux: &fluxv1alpha1.FluxInstallation{
+				Namespace: ptr.To("flux-system"),
+			},
+		}
+		cluster := &extensions.Cluster{
+			Shoot: &gardencorev1beta1.Shoot{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: shootName,
+				},
+				Status: gardencorev1beta1.ShootStatus{
+					TechnicalID:     technicalID,
+					ClusterIdentity: &clusterIdentity,
+				},
+			},
+		}
+		configMap := &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      shootInfoConfigMapName,
+				Namespace: *config.Flux.Namespace,
+			},
+		}
```

Comment:
Initializing the variables typically happens in a `BeforeEach` block. Then, you can reuse the setup for multiple test cases.

Reply:
I omited this block because we have only one testcase here. But since you requested another testcase this will be moved to a BeforeEach block anyway.